SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -10,34 +12,109 @@
 def cust_2017(process):
     process=customisePostLS1(process)
     process=customisePhase1Tk(process)
-    #process=customise_HcalPhase0(process)
-    #process=fixRPCConditions(process)
+    # process=customise_HcalPhase0(process)
+    # process=fixRPCConditions(process)
     return process
 
 def cust_2019(process):
     process=customisePostLS1(process)
     process=customisePhase1Tk(process)
     process=customise_HcalPhase1(process)
-    process=fixRPCConditions(process)
+    # process=fixRPCConditions(process)
     return process
 
+# --> to delete
+# def cust_2023_MuonOnly(process):
+#     process=customisePostLS1(process)
+#     process=customisePhase1Tk(process)
+#     process=customise_HcalPhase1(process)    ### will only work for Geometry Extended2023, but not for Extended2023Muon, because HcalPhase1 collides with ME0
+#     process=customise_gem2019(process)       ### difference between gem2019 and gem2023 is that in the latter special L1 trigger configs need to be loaded
+#     process=fixRPCConditions(process)
+#     process=fixDTAlignmentConditions(process)
+#     process=digiOnlyMuonDetectors(process)
+#     process=recoOnlyMuonDetectors(process)
+#     return process
+
+
 def noCrossing(process):
     process=customise_NoCrossing(process)
     return process
 
 
 def fixRPCConditions(process):
+    process.simMuonRPCDigis.digiModel = cms.string('RPCSimAverageNoiseEffCls')
     if not hasattr(process.GlobalTag,'toGet'):
         process.GlobalTag.toGet=cms.VPSet()
     process.GlobalTag.toGet.extend( cms.VPSet(
         cms.PSet(record = cms.string("RPCStripNoisesRcd"),
-                 tag = cms.string("RPCStripNoise_upscope_mc_v2"),
+                 tag = cms.string("RPC_testCondition_192Strips_mc"),
                  connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_31X_RPC")
+                 ),
+        cms.PSet(record = cms.string("RPCClusterSizeRcd"),
+                 tag = cms.string("RPCClusterSize_PhaseII_mc"),
+                 connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_36X_RPC")
                  )
         )
-                                    )
+    )
     return process
 
+def fixDTAlignmentConditions(process):
```

Comment:
indeed, to fix the runGEMRecHitProducer_cfg.py. It will go in a separate pull request